extension/dnsmasq/dnsmasq.js

```diff
@@ -1282,4 +1282,22 @@ module.exports = class DNSMASQ {
       }
     }
   }
+
+  async cleanUpLeftoverConfig(){
+    try{
+      const userConfigFolder = f.getUserConfigFolder(),
+            dnsConfigFolder = `${userConfigFolder}/dns`,
```

Comment:
Clean up whole 'dns' folder may affect other functions based on dnsmasq, e.g., ad-block, policy filter. It's better to cleanup only safe-search specific files in dns folder.